docs/src/pages/index.tsx

```diff
@@ -1,37 +1,259 @@
+import WebGpuLogoLight from '@site/static/img/webgpu-light.svg';
+import { useColorMode } from '@docusaurus/theme-common';
 
 function HomepageHeader() {
-  const { siteConfig } = useDocusaurusContext();
   return (
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
-        <Heading as="h1" className="hero__title">
-          {siteConfig.title}
-        </Heading>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link className="button button--secondary button--lg" to="/docs/intro">
-            Docs
-          </Link>
+        <div className="row">
+          <div className="col col--6">
+            <div className="container">
+              <Heading as="h2" className="hero__title">
+                The best solution for programmatic live video compositing.
+              </Heading>
+              <p>
+                Process live video sources in real-time. Apply effects, combine videos, or add
+                transitions to produce any output you want.
+              </p>
+              <div className="row">
+                <Link
+                  className="button button--secondary button--lg col col-6 margin--sm"
+                  to="/docs/intro">
+                  Docs
+                </Link>
+                <Link
+                  className="button button--secondary button--outline button--lg col col-6 margin--sm"
+                  to="https://github.com/membraneframework/video_compositor">
+                  View on GitHub
+                </Link>
+              </div>
+            </div>
+          </div>
+          <div className="col col--6">Image</div>
         </div>
       </div>
     </header>
   );
 }
 
+function ProsCards() {
+  const wgpuLink = (
+    <Link href="https://github.com/gfx-rs/wgpu">
+      <code>wgpu</code>
+    </Link>
+  );
+  return (
+    <div className="container">
+      <div className="row">
+        <ProsCard title="Simple Declarative API" icon={FaCode}>
+          <p className="padding--md">
+            Simple Web-like component tree. Easy to pick up for anyone familiar with web
+            development.
+          </p>
+        </ProsCard>
+        <ProsCard title="Language agnostic" icon={FaLink}>
+          <p className="padding--md">
+            The entire configuration process happens over HTTP API, and streams are delivered over
+            RTP. You can leverage technology of your choice and easily integrate it with your own
+            solutions.
+          </p>
+        </ProsCard>
+        <ProsCard title="Hardware accelerated" icon={FaMicrochip}>
+          <p className="padding--md">
+            Rendering is GPU accelerated using {wgpuLink} library, which implements API based on the
+            WebGPU standard. Parts of the standard, like the ability to run WGSL shaders, are
+            directly accessible via API.
```

Comment:
Rendering is GPU accelerated using Rust {wgpuLink} library, based on the WebGPU standard. 
Low-level integrations, like custom WGSL shaders, are directly accessible via LiveCompositor API.

Reply:
>Rendering is GPU accelerated using Rust {wgpuLink} library, based on the WebGPU standard.

I don't think this sentence is clear, what is based on WebGPU? rendering? wgpu?

> Low-level integrations

shaders are not low level parts of the standard, also not sure what integrations here is supposed to mean